src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -406,4 +408,46 @@ public function __clone()
     {
         throw Exception\Uncloneable::fromClass(__CLASS__);
     }
+
+    /**
+     * Retrieves the body of this function as AST nodes
+     *
+     * @return array
+     */
+    public function getBodyAst()
+    {
+        return $this->node->stmts;
+    }
+
+    /**
+     * Retrieves the body of this function as code.
+     *
+     * If a PrettyPrinter is provided as a paramter, it will be used, otherwise
+     * a default will be used.
+     *
+     * Note that the formatting of the code may not be the same as the original
+     * function. If specific formatting is required, you should provide your
+     * own implementation of a PrettyPrinter to unparse the AST.
+     *
+     * @param PrettyPrinterAbstract|null $printer
+     * @return string
+     */
+    public function getBodyCode(PrettyPrinterAbstract $printer = null)
```

Comment:
Is this one actually needed? I wouldn't pretty-print: the consumer can do it, no?